tests/test_api.py

```diff
@@ -18,7 +20,7 @@
 class ApiTests(TestCase):
```

Comment:
ideally we should rename this to `ApiJobTests` so the schema matches with the other `ApiTagTests`